src/tests/Interop/COM/NativeServer/Servers.cpp

```diff
@@ -132,7 +132,7 @@ namespace
             nullptr,
             0,
             REG_SZ,
-            reinterpret_cast<const BYTE*>(fullPath),
+            reinterpret_cast<const uint8_t*>(fullPath),
             static_cast<DWORD>(::TP_slen(fullPath) + 1) * sizeof(fullPath[0]));
```

Comment:
Do we want to also switch `DWORD`? There are a couple instances spread around the interop tests.

Reply:
I'm going to defer `DWORD` for now. I agree this should be explored in the future.